modules/ModuleBuilder/tpls/layoutView.tpl

```diff
@@ -78,7 +78,7 @@
         <div class='le_field' id='{$idCount}'>
             {if ! $fromModuleBuilder && ($col.name != '(filler)')}
                 {capture assign="otherAttributes"}class="le_edit" style="float:right; cursor:pointer;" onclick="editFieldProperties('{$idCount}', '{$col.label}');"{/capture}
-                {sugar_getimage name="edit_inline" ext=".gif" other_attributes=$otherAttributes}
```

Comment:
Don't require these changes as the the edit_inline has been included in the SuiteP theme (if we added this in then it would break the previous themes)